client/src/components/Personal/Personal.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import { reduxForm, Field } from 'redux-form';
+import validate from '../utils/ValidateWorkflow';
+import FormFields from './FormFields';
+import _ from 'lodash';
+import { fetchUser } from '../../actions/index';
+import { connect } from 'react-redux';
+import * as actions from '../../actions/index';
+import { withRouter } from 'react-router-dom';
+import { ToastContainer, toast } from 'react-toastify';
+import { Link } from 'react-router-dom';
+
+class Personal extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            originalForm: null, // Null at initial state. We wait for redux-form to give us the second value
+            firstRender: true
+        }
```

Comment:
We have two states:

1) originalForm: this is the ORIGINAL form upon loading the profile. We want to hold on to this locally so that *if* the user makes modifications to the form and presses "Update", we'll update. If there were no modifications, we will not update.
2) firstRender will be the boolean check to do the above.